lib/trailblazer/operation/controller/active_record.rb

```diff
@@ -7,6 +7,10 @@ def setup_operation_instance_variables!
   end
 
   def operation_model_name
+    # set the right variable name if collection
+    if @model.class.to_s.match(/ActiveRecord_Relation\z/)
```

Comment:
I learned that rails sux and create a fuckin' `Model_ActiveRecord_Relation` class that I can't compared with anything, I need to regexp if it ends with `ActiveRecord_Relation`, please point me a better way of doing this :crying_cat_face: